src/packages/dwlib/dwlib.cc

```diff
@@ -42,6 +45,9 @@ void query_multiple_short(svalue_t *arg, const char *type, int no_dollars, int q
   object_t *ob;
   char *fun;
 
+  //mdw--old code dern't work, copying this from elsewhere.
+  auto max_string_length = CONFIG_INT(__MAX_STRING_LENGTH__);
```

Comment:
I'm assuming by changing MAX STRING LENGTH we can change the max length of posts?
Or is this a base C variable? 
I'll look into this as kitty wants bigger posts.